frontend/src/__tests__/cypress/cypress/pages/modelRegistry.ts

```diff
@@ -6,8 +6,8 @@ class ModelRegistry {
     this.waitLanding();
   }
 
-  visit(modelRegistry?: string) {
-    cy.visit(`/modelRegistry${modelRegistry}`);
+  visit() {
+    cy.visitWithLogin(`/modelRegistry`);
```

Comment:
To tackle a cypress test flake , we are now using  `visit()` instead of `visitWithLogin()`
```suggestion
    cy.visit(`/modelRegistry`);
```